package.json

```diff
@@ -22,7 +22,8 @@
     "generate:ssl-private-key": "openssl genrsa -out ./key.pem",
     "import:sample-data": "ts-node ./src/utilities/loadSampleData.ts",
     "import:sample-data:prod": "node ./build/utilities/loadSampleData.js",
-    "gen:schema": "graphql-inspector introspect ./src/typeDefs/**/**/*.ts --write ./schema.graphql "
+    "gen:schema": "graphql-inspector introspect ./src/typeDefs/**/**/*.ts --write ./schema.graphql ",
+    "generate:toc": "find . -maxdepth 1 -name '*.md' -exec markdown-toc -i {} \\;"
```

Comment:
Will this now be run automatically as a githook for commits?

Reply:
Currently it is not. But if you say then I'll add.

Reply:
Now that this option has been created, adding it to a githook will be a welcome addition.